src/ring_theory/local_properties.lean

```diff
@@ -99,10 +100,43 @@ if `P` holds for `R →+* S` whenever there exists a set `{ r }` that spans `R`
 Note that this is equivalent to `ring_hom.of_localization_finite_span` via
 `ring_hom.of_localization_span_iff_finite`, but this has less restrictions when applying. -/
 def ring_hom.of_localization_span :=
-  ∀ {R S : Type u} [comm_ring R] [comm_ring S] (f : by exactI R →+* S)
+  ∀ ⦃R S : Type u⦄ [comm_ring R] [comm_ring S] (f : by exactI R →+* S)
     (s : set R) (hs : by exactI ideal.span s = ⊤)
     (H : by exactI (∀ (r : s), P (localization.away_map f r))), by exactI P f
 
+/-- A property `P` of ring homs satisfies `ring_hom.localization_away_is`
+ if `P` holds for each localization map `R →+* Rᵣ`. -/
+def ring_hom.localization_away_is : Prop :=
```

Comment:
I find the `is` part of the name too confusing with equality, how about a name like:
```suggestion
def ring_hom.holds_for_localization_away : Prop :=
```